<launch>

  <!-- Load joint controller configurations from YAML file to parameter server -->
  <rosparam file="$(find ThrowBot)/config/gp8_gazebo_controller.yaml" command="load"/>

  <!-- load the controllers -->
  <node name="motoman_gp8_controller_spawner" pkg="controller_manager"
		type="spawner" output="screen"  ns = "/motoman_gp8"
		args="joint_state_controller
			  gp8_controller"/>

  <!-- convert joint states to TF transforms for rviz, etc -->
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" ns = "/motoman_gp8"
		respawn="false" output="screen">
  </node>

</launch>